<launch>

  <!-- CAN realated  -->
  <arg name="baudrate" default='1000000' />
  <arg name="channel" default="can0" />
  <arg name="delay_0" default="0.5" />

  <arg name="channel_can" default="can0" />

  <param name="/baudrate"  value="$(arg baudrate)" /> 

  <param name='/channel_can' value="$(arg channel) "/>
  <param name='/delay_0' value="$(arg delay_0) "/>

  <param name ='/bustype' value='socketcan'/>
  <param name='/arbitration_id' value='0x188'/>

    <!-- end of CAN related -->



  <node name="subsciber_for_mobile" pkg="demo_customer" type="ros_mobile_can.py" output="screen" />
  <node pkg='turtlesim' type='turtlesim_node' name='turtlesim_node'/>

</launch>

